src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioDocumentTracker.cs

```diff
@@ -189,5 +203,17 @@ internal void EditorSettingsManager_Changed(object sender, EditorSettingsChanged
         {
             OnContextChanged(_project, ContextChangeKind.EditorSettingsChanged);
         }
+
+        internal void Imports_Changed(object sender, ImportsChangedEventArgs args)
```

Comment:
Why internal? Should also rename to `Import_Changed` and the args should probably be `ImportChangedEventArgs`

Reply:
For testing. I'll add the comment